Packs/CrowdStrikeIntel/Integrations/CrowdStrikeFalconIntel_v2/CrowdStrikeFalconIntel_v2.yml

```diff
@@ -26,6 +26,20 @@ configuration:
   - low
   - medium
   - high
+- additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
```

Comment:
Was not sure what default value to use

Reply:
You can ask Dror or Danil

Reply:
Spoke to Dror, we agreed to put `defaultvalue: C - Fairly reliable`